package.json

```diff
@@ -15,7 +15,7 @@
     "build": "./build.sh",
     "lint": "eslint --ext .js,.jsx .",
     "set-up": "npm run build",
-    "server": "node server.js",
+    "server": "node --experimental-modules --es-module-specifier-resolution=node server.js",
```

Comment:
Hmm, as someone who routinely runs `node server.js` directly, I'd love if there was a way to enable these options/features another way. Are `package.json` options doable?

Reply:
I think it either has to be passed as flags directly or in the `NODE_OPTIONS` env var. Some quick internet searching along the lines of "set node options in package.json" makes me think that there isn't a good solution here if you still want to use `node server.js`. See https://github.com/nodejs/node/issues/37409 for some related discussion.

Reply:
Ah, hmm. I see.

Do we really need `--experimental-modules`? It's [marked as deprecated](https://nodejs.org/docs/latest-v14.x/api/cli.html#cli_experimental_modules) and seems to be a historical thing replaced now by `"type": "module"` in `package.json`.

If we wanted to avoid needing `--es-module-specifier-resolution=node`, we'd just have to update the imports to use `.js` and `…/index.js`? This doesn't seem so burdensome to me and arguably makes the mapping of code to files more explicit/clear.

Relatedly, `Procfile` will also need updating if we do stick with required extra `node` options.

Reply:
Good points, I removed both and added f1f1f972b14f38012ef6c32650b369eed445a6f5 to resolve this.